xml/System.IO.IsolatedStorage/IsolatedStorageFileStream.xml

```diff
@@ -1997,10 +2011,10 @@ Dim source As New IsolatedStorageFileStream(UserName,FileMode.Open,isoFile)
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">The read-only byte memory from which to copy bytes to the current isolated storage file stream.</param>
+        <param name="cancellationToken">The token to monitor for cancellation requests. The default value is <see cref="P:System.Threading.CancellationToken.None" />.</param>
+        <summary>Asynchronously writes a block of bytes to the isolated storage file stream object using data read from the buffer.</summary>
```

Comment:
Here again, it would be good to disambiguate the two overloads -- perhaps "a buffer consisting of a byte array" and "a buffer consisting of a read-only byte memory range"?

Reply:
Ok, I'll do it separately from the suggestions batch.

Reply:
```suggestion
        <summary>Asynchronously writes a block of bytes to the isolated storage file stream object using data read from a buffer consisting of a read-only byte memory range.</summary>
```